server/src/WflowServer.jl

```diff
@@ -0,0 +1,9 @@
+module WflowServer
+using ZMQ
+using JSON3
+using StructTypes
+using Wflow
```

Comment:
Switching to `import X` or `using X: a, b` will make it more future proof in case these packages start to export clashing names at some point.

Reply:
Good point, I agree to switch this, see https://github.com/Deltares/Wflow.jl/pull/356/commits/0104c187adb5a656906aeae6318fb44dad68b6f6.